packages/indexer/native/src/lib.rs

```diff
@@ -110,6 +122,92 @@ impl NativeIndexer {
         }
         Ok(())
     }
+
+    fn publish_new_block_info(&self, block: &CoreBlockView) {
+        let emitters = self.emitters.read().unwrap();
+        let lock_script_emitters = emitters.iter().filter(|x| x.lock.is_some());
+        let type_script_emitters = emitters.iter().filter(|x| x.type_.is_some());
+
+        let transactions = block.transactions();
+        for (_tx_index, tx) in transactions.iter().enumerate() {
+            // TODO: cell_consumed event ?
```

Comment:
This is production code, unless absolutely necessary, we shouldn't leave any TODOs.

Reply:
updated.